tools/clang/test/CodeGenSPIRV/type.matrix.majorness.zpc.hlsl

```diff
@@ -0,0 +1,33 @@
+// Run: %dxc -T ps_6_0 -E main -Zpc
+
+// CHECK: OpDecorate %_runtimearr_mat2v3float ArrayStride 24
+// CHECK: OpMemberDecorate %type_StructuredBuffer_mat2v3float 0 RowMajor
+       StructuredBuffer<float2x3> ROSB1;
+// CHECK: OpDecorate %_runtimearr_mat3v2float ArrayStride 32
+// CHECK: OpMemberDecorate %type_RWStructuredBuffer_mat3v2float 0 RowMajor
+     RWStructuredBuffer<float3x2> RWSB1;
+// CHECK: OpDecorate %_runtimearr_mat4v3float ArrayStride 48
+// CHECK: OpMemberDecorate %type_AppendStructuredBuffer_mat4v3float 0 RowMajor
+ AppendStructuredBuffer<float4x3> ASB1;
+// CHECK: OpDecorate %_runtimearr_mat3v4float ArrayStride 64
+// CHECK: OpMemberDecorate %type_ConsumeStructuredBuffer_mat3v4float 0 RowMajor
+ConsumeStructuredBuffer<float3x4> CSB1;
+
+// NOTE: The parsed AST does not convey the majorness information for
```

Comment:
😢

Reply:
fxc disallows such usage, so it won't be a problem immediately. I've raised the question in #1032.